pkg/controllers/provisioning/binpacking/packer.go

```diff
@@ -182,7 +182,10 @@ func (p *Packer) packWithLargestPod(unpackedPods []*v1.Pod, packables []*Packabl
 			// Trim the bestInstances so that provisioning APIs in cloud providers are not overwhelmed by the number of instance type options
 			// For example, the AWS EC2 Fleet API only allows the request to be 145kb which equates to about 130 instance type options.
 			for j := i; j < len(packables) && j-i < MaxInstanceTypes; j++ {
-				bestInstances = append(bestInstances, packables[j])
+				// Excluding instance type with smaller memory
+				if packables[i].Memory().Cmp(*packables[j].Memory()) != 1 {
```

Comment:
I typically see this written in the form:
```
<= 0  // a <= b
>= 0 // a >= b
== 0 // a == b
< 0  // a < b 
> 0 //  a > b
```

Reply:
done